recipes/default.rb

```diff
@@ -18,9 +18,7 @@
 #
 
 # jenkins 1.612 and above requires java 7
-node.default['java']['jdk_version'] = '8'
-
-include_recipe 'java'
+openjdk_pkg_install '11'
```

Comment:
Let's not upgrade this to 11 quite yet. Can we please keep this at 8 for now?

Reply:
Sure, my bad!